docs/topics/debug-flow-with-idea.md

```diff
@@ -102,6 +102,13 @@ Create a Kotlin [flow](https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-cor
 
     ![Debug the coroutine](flow-debug-2.png)
 
+> You can disable a "was optimised out" feature via the `-Xdebug` compiler option.
+> The "was optimised out" feature optimizes variables when you use suspend functions.
+> However, debugging a code with optimized variables is hard because you don't see their values.
+> Be careful: disabling this feature via `-Xdebug` can [cause memory leaks](https://youtrack.jetbrains.com/issue/KT-48678/Coroutine-debugger-disable-was-optimised-out-compiler-feature#focus=Comments-27-6015585.0-0).
```

Comment:
I think stronger wording is warranted in this case. Like, writing in bold, "**never use this flag in production, only for debugging**." Specific analytics showed that people are *really* unwilling to read the documentation and will, in most cases, grab the first thing that seems correct, put it into their build files, and walk away happy.